bundles/http/Citrix/Microsoft-Outlook/script.js

```diff
@@ -0,0 +1,319 @@
+						name: 'location_display_name',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'online_meeting_join_url',
+						type: 'STRING',
+						length: 500
+					},
+					{
+						name: 'online_meeting_provider',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'organizer_email_address_a',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'organizer_email_address_n',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'original_start_time_zone',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'series_master_id',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'start_date_time',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'subject',
+						type: 'STRING',
+						length: 255
+					}
+				]
+			},
+			{
+				name: 'calender_view_attendees',
+				columns: [
+					{
+						name: 'unique_id',
+						type: 'STRING',
+						length: 255,
+						primaryKey: true
+					},
+					{
+						name: 'parent_i_cal_u_id',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'root_i_cal_u_id',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'email_address_address',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'email_address_name',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'type',
+						type: 'STRING',
+						length: 255
+					}
+				]
+			},
+			{
+				name: 'my_events',
+				columns: [
+					{
+						name: 'i_cal_u_id',
+						type: 'STRING',
+						length: 255,
+						primaryKey: true
+					},
+					{
+						name: 'id',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'recurrence_pattern_day_of',
+						type: 'INTEGER'
+					},
+					{
+						name: 'recurrence_pattern_type',
+						type: 'STRING',
+						length: 255
+					},
+					{
+						name: 'recurrence_range_end_date',
+						type: 'DATE'
+					}
+				]
+			}
+		],
+		relationships: [
+			{
+				name: 'nested_table_1',
+				primaryTable: 'calender_view',
+				foreignTable: 'calender_view_attendees',
+				columnPairs: [
+					{
+						primaryKey: 'i_cal_u_id',
+						foreignKey: 'parent_i_cal_u_id'
+					}
+				]
+			},
+			{
+				name: 'fk_my_events_id',
+				primaryTable: 'calender_view',
+				foreignTable: 'my_events',
+				columnPairs: [
+					{
+						primaryKey: 'series_master_id',
+						foreignKey: 'id'
+					}
+				]
+			},
+			{
+				name: 'fk_calender_v_organizer_e',
+				primaryTable: 'users',
+				foreignTable: 'calender_view',
+				columnPairs: [
+					{
+						primaryKey: 'user_principal_name',
+						foreignKey: 'organizer_email_address_a'
+					}
+				]
+			}
+		]
+	}
+});
+
+const moment = library.load('moment-timezone');
+const uuid = library.load('uuid');
+const startDate = moment.utc().subtract(1, 'd').format();
+const endDate = moment.utc().add(30, 'd').format();
+const pageSize = 100;
+let userId = [];
```

Comment:
```suggestion
```
Declaring global parameters might lead to wrong output, its better to define the variables within the function itself.

Reply:
I need to pass the `pageSize` to all the functions so keeping `pageSize` as a global declaration & moving rest all to the function.